MonoGame.Framework/Graphics/Shader/ConstantBuffer.cs

```diff
@@ -61,8 +61,12 @@ internal void Clear()
             PlatformClear();
         }
 
-        private void SetData(int offset, int rows, int columns, object data)
```

Comment:
@SimonDarksideJ @yousernaym 

So there is nothing wrong here.  It is a `private` method internal to MG.  We don't need to create a new method that will never get called.